src/components/20-molecules/accordeon/pw.ui.test.js

```diff
@@ -0,0 +1,54 @@
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-accordeon';
+
+describe('Accordeon', () => {
+  it('Should check if accordeon is open', async () => {
+    await openAccordeon();
+
+    expect(
+      await page.$eval('.m-accordeon__content', el =>
+        el.classList.contains('m-accordeon__content--open')
```

Comment:
This test would still pass, if I remove all CSS rules within that CSS class.

It's better to check for direct properties instead of classes. For instance, max-height or whatever concludes  `open`.

Reply:
I think it makes sense to check if the `&--open` class is available but in addition I've added another expect to check the max-height